client/onboarding/steps/loading.tsx

```diff
@@ -61,7 +65,7 @@ const LoadingStep: React.FC< Props > = () => {
 			isEligible = await isEligibleForPo();
 		} catch ( error ) {
 			// fall back to full KYC scenario.
-			// TODO maybe log these errors in future, e.g. with tracks.
+			trackEligibilityError( error );
```

Comment:
Totally agree with you! I have recently added this to the P2 next steps section

Reply:
I agree with this comment - I also wonder if we should consider whether to have a more generic Tracks event for errors (e.g. `wcpay_client_error`) and then have a prop for specific error codes (e.g. `type` = `onboarding_flow_eligibility_error`) or if it's better to do as you have done and have specific error events for specific errors.

The benefit to it being more generic is that it may make monitoring and maintaining easier, but I guess seeing how other divisions have used this functionality would be a good guide here.